rest-api-spec/src/main/resources/rest-api-spec/test/search/10_source_filtering.yaml

```diff
@@ -87,7 +86,7 @@
           body:
             fields: [ include.field2, _source ]
             query: { match_all: {} }
-  - match: { hits.hits.0.fields: { include.field2 : [v2] }}
+  - match: { hits.hits.0._source.include.field2: v2 }
   - is_true:  hits.hits.0._source
```

Comment:
The point of the two above tests seems to be to test bw compat, I think we can remove them?